exonum/src/api/public/system.rs

```diff
@@ -52,30 +53,36 @@ impl SystemApi {
         }
     }
 
-    fn mempool_info(&self) -> MemPoolInfo {
-        MemPoolInfo { size: self.pool.read().expect("Expected read lock").len() }
+    fn mempool_info(self, router: &mut Router) {
+        let mempool = move |_: &mut Request| -> IronResult<Response> {
+            let info = MemPoolInfo { size: self.pool.read().expect("Expected read lock").len() };
+            self.ok_response(&::serde_json::to_value(info).unwrap())
```

Comment:
@DarkEld3r how did you missed `::...` syntax here? :D It think it's better to import `to_value` into scope

Reply:
As for me, `to_value` functions itself very non-descriptive. I prefer the following:
```rust
use serde_json;

self.ok_response(&serde_json::to_value(info).unwrap())
```